cluster/gce/templates/common.sh

```diff
@@ -44,17 +42,26 @@ install-salt() {
     salt-common_2014.1.13+ds-1~bpo70+1_all.deb
     salt-minion_2014.1.13+ds-1~bpo70+1_all.deb
   )
-  if [[ ${1-} == '--master' ]]; then
-    TARS+=(salt-master_2014.1.13+ds-1~bpo70+1_all.deb)
-  fi
   URL_BASE="https://storage.googleapis.com/kubernetes-release/salt"
 
   for tar in "${TARS[@]}"; do
     download-or-bust "${URL_BASE}/${tar}"
-    dpkg -i "${tar}"
+    dpkg -i "${tar}" || true
   done
 
   # This will install any of the unmet dependencies from above.
   apt-get install -f -y
+}
 
+# Ensure salt-minion *isn't* running
+stop-salt-minion() {
+  # This ensures it on next reboot
+  echo manual > /etc/init/salt-minion.override
+
+  service salt-minion stop
+  until ! service salt-minion status >/dev/null; do
```

Comment:
I think I had `until` on the brain from the curl loops in the startup script. :)